airbyte-integrations/connectors/source-stripe/tools/README.md

```diff
@@ -0,0 +1,20 @@
+This is a tool to seed records into a test Stripe account.
```

Comment:
Sidenote: I feel we should have this for all the sources we have so that if the env has a problem or gets deleted, we can just scrap everything and start again. Could be a postman script too, the implementation does not matter much

Reply:
💯 
This has been discussed many times but we always punted on it. This can serve as a proof of feasibility